### ISSUE TITLE
"Skip to main content" link is missing translation [S]

### ISSUE THREAD

**Issue by brdunfield** [cds-snc/notification-api#970]
cc @anikbrazeau

**Comment by anikbrazeau** [comment]
**EN** "Skip to main content" 
**FR** "Passer au contenu principal"
